assets/i18n/en.json

```diff
@@ -1,5 +1,6 @@
 {
   "I18N_ABOUT_PAGE_TITLE": "About us - Oppia",
+  "I18N_ACTION_APPLY_TO_TEACH_WITH_OPPIA": "Apply to Teach With Oppia",
```

Comment:
@seanlip sure, here it goes:
If you want "Teach with Oppia" translated:
"I18N_ACTION_APPLY_TO_TEACH_WITH_OPPIA": "Mendaftar ke Mengajar Dengan Oppia",

If you want "Teach with Oppia" **not** translated:
"I18N_ACTION_APPLY_TO_TEACH_WITH_OPPIA": "Mendaftar ke Teach With Oppia",